apps/myaccount/src/components/overview/widgets/profile-widget.tsx

```diff
@@ -129,7 +132,13 @@ export const ProfileWidget: FunctionComponent<ProfileWidgetPropsInterface> = (
                 className="overview"
                 data-testid={ `${testId}-settings-section` }
                 header={ t("myAccount:components.overview.widgets.profileStatus.header" ,
-                    { productName: config.ui.productName }) }
+                    { 
+                        productName: (organizationDetails?.displayName || 
+                            !isEmpty(organizationDetails?.displayName)) 
+                            ? organizationDetails?.displayName 
+                            : config?.ui?.productName 
+                    }
+                ) }
```

Comment:
`!isEmpty(displayName) ? displayName : config?.ui?.productName;`
Shall we simplify the logic like this?